### ISSUE TITLE
It is time to drop the "(formerly Perl 6)"

### ISSUE THREAD

**Issue by lizmat** [Raku/problem-solving#377]
I think it's time to ask authors of blog posts and web sites to remove the "(formerly Perl 6)" or "(née Perl 6)" from any mention of the Raku Programming Language.

**Comment by coke** [comment]
I think we're in a good spot on this for raku/doc, but doing a review as part of this ticket is probably a good idea.

**Comment by niner** [comment]
I'm not sure about this. I still meet people with whom it helps to mention Perl 6 and with those there's no downside to doing so.

**Comment by lizmat** [comment]
> I still meet people with whom it helps to mention Perl 6 and with those there's no downside to doing so.

I'm not saying nobody should mention it anymore.  I'm saying that website and blog posts shouldn't mention it anymore in the first introduction.  It's just not a good first impression.

**Comment by 2colours** [comment]
Fair point but is there an "action plan", even if just contacting the people who regularly blog about Raku ~~neé Perl 6~~? The actual technical renaming of things (especially wrt extensions) didn't turn out to be very influential; I wouldn't be surprised if most of the code still had pm6, p6 kind of extensions.

(By the way, indeed there are contexts when you just want to "ring a bell" rather than, say, earn a user, and then the connection to Perl can still help...)

**Comment by jubilatious1** [comment]
This is converse bike-shedding. Let's fix some bugs!

**Comment by tbrowder** [comment]
I recently introduced a Debian member who knows Python well (as in Mailman 3) to Raku as “formerly known as Perl 6” and he said that was the first he had heard of Raku. So I agree that it is a good bridge to Raku for some folks.
